src/Meilisearch/DocumentQuery.cs

```diff
@@ -16,8 +16,8 @@ public class DocumentQuery
         public int? Limit { get; set; }
 
         /// <summary>
-        /// Gets or sets the attributes to retrieve.
+        /// Gets or sets the fields.
         /// </summary>
-        public string AttributesToRetrieve { get; set; }
+        public string Fields { get; set; }
```

Comment:
I strongly think this should be an array/list. As a user, it will be easier to send the data to the SDK if it will be an array, and if Meili changes the format of the separator, we can ensure it will be properly change :)